index.html

```diff
@@ -4,22 +4,22 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Ecom-Web</title>
+    <title>Fashion-Mart</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <ul class="container">
         <li class="pages-list">
-           <a class="anchors" href="/HTML and CSS/home.html">Home Page</a>
+           <a class="anchors" href="/screens/produtlist/product.html">Home Page</a>
```

Comment:
I think the class name should be `anchor` as it's a single anchor tag.

Reply:
thanks updated